lib/auth/bot.go

```diff
@@ -262,6 +263,18 @@ func (s *Server) getBotUsers(ctx context.Context) ([]types.User, error) {
 	return botUsers, nil
 }
 
+// supportedBotJoinMethods should match SupportedJoinMethods declared in
```

Comment:
It'd be nice to have these eventually share the same variable - but I don't want to have a dependency on `lib/tbot` from `lib/auth` at this moment - I'll have a further think about this, but this is at least better than before.

Reply:
Yeah - I think that seems more sensible, or even possibly exporting it from `api`. I'd prefer to keep that out of the scope of this, as once I started pulling this thread, there's potentially some other code I'd like to move around too.